StPetersburg/InEtn40556.xml

```diff
@@ -0,0 +1,223 @@
+                    <title type="complete" ref="LIT5075MagicPrayer"><certainty locus="value" match="../@ref" cert="medium"></certainty></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"></gap> ጸሎት፡ በእንተ፡ ሕማመ፡ ባርያ፡ ወሌጌዎን፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i5">
+                    <title type="incomplete" ref="LIT2709Matthew"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>8.28-32</note>
+                  </msItem>
+                  <msItem xml:id="ms_i6">
+                    <title type="incomplete" ref="LIT2711Mark"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>1.23-28</note>
+                  </msItem>
+                  <msItem xml:id="ms_i7">
+                    <title type="incomplete" ref="LIT2713Luke"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>8.43-44</note>
+                  </msItem>
+                  <msItem xml:id="ms_i8">
+                    <title type="complete" ref="LIT2765RepCh49"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">ሰላም፡ ለከ፡ ሰዳዴ፡ ሰይጣናት፡</incipit>
+                  </msItem>
+
+                </msContents>
+                <physDesc>
+                  <objectDesc form="Scroll">
+                      <supportDesc>
+                          <support>
+                              <material key="parchment"/>
+                              <note>The parchment is described as thick in <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                              <citedRange unit="page">220</citedRange></bibl>.</note>
+                          </support>
+                          <extent>
+                              <measure unit="leaf">3</measure>
+                              <dimensions type="outer" unit="mm">
+                                <height>2080</height>
+                                <width>70</width>
+                              </dimensions>
+                          </extent>
+
+                      </supportDesc>
+                      <layoutDesc>
+                          <layout columns="1" writtenLines="300">
+                          </layout>
+                      </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                    <handNote script="Ethiopic" xml:id="h1">
+                        <seg type="ink">Black, red</seg>
+                        <seg type="script">The script is archaic, angular according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">221</citedRange></bibl>.</seg>
+                        <date><date>Second half of the 18th-cent - beginning of the 19th century</date> according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
```

Comment:
There is a double  `<date><date>` :) And it could be useful to encode the date within `<date>`

Reply:
I think, this is what I did, there are two date-elements there. Or I do not get the point

Reply:
What I see here is:
` <date><date>Second half of the 18th-cent - beginning of the 19th century</date> according to <bibl><ptr target="bm:Chernetsov1975Svitki"/><citedRange unit="page">221</citedRange></bibl>.</date>`
There are two date elements, which is a bit redundant, but the date itself is not marked up precisely:
`<date notBefore="1750" notAfter="1849">` or whichever dates seem best to you.